decompressed/base/lib/mount_modroot/05_transfer_basefiles

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+upperdir=$1
+overlay_bank=/overlay/bank_2
+
+preserve_base_file() { # <source> <dest>
+    local source=$1
+    local dest=$2
+
```

Comment:
```bash
update_file_if_needed() { # <file> <source> <dest>
    cmp -s $2$1 $3$1 || cp -a $2$1 $3$1
}

preserve_list="/etc/init.d/rootdevice /etc/rc.d/S94rootdevice /usr/sbin/random_seed /sbin/mount_root-mod /lib/mount_modroot/*"

for f in $preserve_list; do
    mkdir -p $dest$(dirname $f)
    update_file_if_needed $f $source $dest
done
```

Reply:
il cmp potremmo anche evitarlo credo, viene controllato in automatico

Reply:
E da chi?

Reply:
non sono convinto sul `cp /lib/mount_modroot/*` e ora non è il caso ma se ci sono sottocartelle potrebbe darci problemi, non so se vale la pena mixare con il find tipo https://github.com/Ansuel/tch-nginx-gui/blob/master/decompressed/upgrade-pack-specificDGA/setup.sh#L7-L19

Reply:
@LuKePicci credo sia dal overlayfs

Reply:
@FrancYescO non possono esserci sottocartelle, si spaccherebbe anche l'include degli stessi

@Ansuel ma qui scrivi direttamente in /overlay quindi non passi da overlayfs

Reply:
Comunque - non ne sono certo ma - credo funzioni anche con sottocartelle

Reply:
direi che nemmeno per le cartelle funziona 😅
<img width="426" alt="image" src="https://user-images.githubusercontent.com/4574593/79808397-fac35280-836d-11ea-916f-5f8e5e833622.png">

Reply:
WTF

Reply:
giustamente perchè cp /roba/* /altraroba/* lo manda in pappa, fai prima ad elencarli tutti

Reply:
si infatti mi son rotto e ho fatto così